src/main/java/picard/sam/markduplicates/UmiAwareDuplicateSetIterator.java

```diff
@@ -151,11 +156,17 @@ private void process(final DuplicateSet set) {
                             throw new PicardException("UMIs of differing lengths were found.");
                         }
                     }
-                    metrics.updateUmiRecordMetrics(currentUmi, inferredUmi);
+//                    metrics.updateUmiRecordMetrics(currentUmi, inferredUmi);
+                    metrics.OBSERVED_BASE_ERRORS += hammingDistance(currentUmi, inferredUmi);
+                    observedUmiBases += currentUmi.length();
+                    observedUmis.increment(currentUmi);
+                    inferredUmis.increment(inferredUmi);
                 }
             }
         }
-        metrics.updateDuplicateSetMetrics(maxEditDistanceToJoin, duplicateSets.size());
+
+        metrics.DUPLICATE_SETS_WITH_UMI += duplicateSets.size();
+        metrics.DUPLICATE_SETS_WITHOUT_UMI++;
```

Comment:
DUPLICATE_SETS_WITHOUT_UMI doesn't quite convey the meaning I think.

don't you mean DUPLICATE_SETS_IGNORING_UMI? or ORIGINAL_DUPLICATE_SETS ?